go/api/base_client.go

```diff
@@ -7290,3 +7480,136 @@ func (client *baseClient) GeoSearchStoreWithInfoOptions(
 		infoOptions,
 	)
 }
+
+// Publish posts a message to the specified channel. Returns the number of clients that received the message.
+//
+// Channel can be any string, but common patterns include using "." to create namespaces like
+// "news.sports" or "news.weather".
+//
+// See [valkey.io] for details.
+//
+// [valkey.io]: https://valkey.io/commands/publish
+func (client *baseClient) Publish(channel string, message string) (int64, error) {
```

Comment:
I propose to move all new commands to another PR(s)

Reply:
Everything but publish has been moved. Publish is used in the existing tests.